src/lib/equinox.ts

```diff
@@ -35,7 +35,7 @@ export function launch(exec: string, address: AddressInfo, application: Applicat
     serverArgs.push('-configuration');
     serverArgs.push(`${application.applicationDir}/${bundle}`);
 
-    const workspace = resolve(application.storageDir.path, 'workspace');
+    const workspace = resolve(application.storageDir.fsPath, 'workspace');
```

Comment:
what's the difference?

Reply:
the path returns / separated path, whereas fsPath returns correct path like c:\path\path